converter/shieldhit/parser.py

```diff
@@ -220,8 +220,8 @@ def _get_material_id(self, uuid: str) -> int:
         """Find material by uuid and retun its id."""
         for idx, item in enumerate(self.geo_mat_config.materials):
             if item[0] == uuid:
-                # If the material is a DEFALUT_MATERIAL then we need the value not its index
-                if item[1] in list(DEFALUT_MATERIALS.values()):
+                # If the material is a DefaultMaterial then we need the value not its index
+                if item[1] in DefaultMaterial._value2member_map_:
```

Comment:
`_value2member_map_` is undocumented Python feature. Add a comment here that you are looking for an enum entry with given value.

Reply:
I've added the comment. Could you change deepsource settings so this doesn't block the PR? I don't have the required rights.

Reply:
This should silence deepsource:
```suggestion
                if item[1] in DefaultMaterial._value2member_map_:  # skipcq: PYL-W0212
```